django/contrib/admin/templates/admin/change_form.html

```diff
@@ -18,7 +18,7 @@
 <a href="{% url 'admin:index' %}">{% translate 'Home' %}</a>
 &rsaquo; <a href="{% url 'admin:app_list' app_label=opts.app_label %}">{{ opts.app_config.verbose_name }}</a>
 &rsaquo; {% if has_view_permission %}<a href="{% url opts|admin_urlname:'changelist' %}">{{ opts.verbose_name_plural|capfirst }}</a>{% else %}{{ opts.verbose_name_plural|capfirst }}{% endif %}
-&rsaquo; {% if add %}{% blocktranslate with name=opts.verbose_name %}Add {{ name }}{% endblocktranslate %}{% else %}{{ original|truncatewords:"18" }}{% endif %}
+&rsaquo; {% if add %}{% blocktranslate with name=opts.verbose_name %}<a href="#" aria-current="page">Add {{ name }}</a>{% endblocktranslate %}{% else %}<a href="#" aria-current="page">{{ original|truncatewords:"18" }}</a>{% endif %}
```

Comment:
We can avoid the duplication and churn to the translation strings:

```suggestion
&rsaquo; <a href="#" aria-current="page">{% if add %}{% blocktranslate with name=opts.verbose_name %}Add {{ name }}{% endblocktranslate %}{% else %}{{ original|truncatewords:"18" }}{% endif %}</a>
```